Application/EdFi.Ods.Api.Startup/Features/Installers/SecurityInstaller.cs

```diff
@@ -269,9 +272,28 @@ private void RegisterIRelationshipsAuthorizationContextDataProviders(IWindsorCon
             // Register all context data providers
             var relationshipContextDataProviderTypes = _assembliesProvider.GetAssemblies()
                 .Where(a => a.IsExtensionAssembly() || a.IsStandardAssembly())
-                .SelectMany(a => a.GetTypes())
-                .Where(
-                    t => !t.IsAbstract && typeof(IRelationshipsAuthorizationContextDataProvider<,>).IsAssignableFromGeneric(t))
+                .SelectMany(a =>
+                {
+                    _logger.Info(
```

Comment:
should this be debug?

Reply:
It's a one time entry at start up, so I felt it had value to be included as INFO. The DEBUG entry includes all the types found as well.